packages/gallery/example/src/index.html

```diff
@@ -0,0 +1,380 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>StormID</title>
```

Comment:
Could we add the viewport meta tag in here?  The examples are not that important, but it created some odd behaviours in some components for the playwright mobile browsers without it.